src/lamplib/README.md

```diff
@@ -1,110 +0,0 @@
-🧞🐍 Genny Python  🧞🐍
-========================
-
-Genny uses Python for glue with other systems and for
-doing pre/post-processing of workloads.
-
-Quick-Start
------------
-
-Genny uses the `.python-version` file to determine
-which version of Python to use. Consider using
-[pyenv](https://github.com/pyenv/pyenv).
-
-```sh
-brew update && brew install pyenv
-pyenv install # uses .python-version to determine python version
-pyenv rehash
-```
-
-Once a suitable python is installed, setup a virtual
-environment and install the `genny` package:
-
-```sh
-pip install virtualenv
-virtualenv venv
-source ./venv/bin/activate
-pip install .
-```
-
-Or if you wish to install and develop the scripts locally at the same time,
-you can use `setup.py develop` to avoid having to install after every change:
-
-```sh
-python setup.py develop
-```
-
-
-Script: `lint-yaml`
----------------------
-
-```sh
-lint-yaml
-```
-
-Lint most of the YAML files in the Genny repository,
-including workloads, phases, and `evergreen.yml`.
-
-Make you run the script from the root of the Genny repository
-(so it can pick up on the `.yamllint` linter config file)
-
-
-Script: `genny-auto-tasks`
```

Comment:
This is the only section that isn't reflected in the general readme. Is this something we should port over? Not sure if anyone is actually running genny-auto-tasks locally.

Reply:
We can kill it. Auto-tasks is a part of `run-genny`.